pkg/discovery/network/openshift4.go

```diff
@@ -43,8 +42,8 @@ func discoverOpenShift4Network(dynClient dynamic.Interface) (*ClusterNetwork, er
 	crClient := dynClient.Resource(openshift4clusterNetworkGVR)
 
 	cr, err := crClient.Get("default", metav1.GetOptions{})
+
 	if err != nil {
-		klog.Info("Attempted network discovery for OpenShift4, no clusternetworks CRD")
```

Comment:
Perhaps we should at least log this to debug (including `err`) rather than silence it completely.

Reply:
In any case, it should be symmetric to all of the discovery mechanism

Reply:
Good comments.

Reply:
Actually we just need to check for `IsNotFound` error. This is addressed as part of https://github.com/submariner-io/submariner-operator/pull/581 which should obviate the need for any changes in this PR.